hosted/helpers/helper_common.go

```diff
@@ -403,10 +403,10 @@ func ContainsString(slice []string, item string) bool {
 	return false
 }
 
-func GetRancherVersions() (string, string, string) {
+func GetRancherVersions(rancherFullVersion string) (string, string, string) {
 	var rancherChannel, rancherVersion, rancherHeadVersion string
```

Comment:
Rename `RancherVersion` in structs.go to `RancherFullVersion`, not to be confused with RancherVersion from here.